SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/GameModeReplicationTest/GameModeReplicationTest.cpp

```diff
@@ -45,18 +45,24 @@ void AGameModeReplicationTest::PrepareTest()
 {
 	Super::PrepareTest();
 
-	check(GetWorld()->GetGameState()->GameModeClass == AGameModeReplicationTestGameMode::StaticClass());
-
 	AuthorityServersCount = 0;
 
-	AddStep(TEXT("Changing replicated value on the authoritative server"), FWorkerDefinition::AllServers, nullptr, [this]() {
+	AddStep(TEXT("Check initial replicated value correct on all server"), FWorkerDefinition::AllServers, nullptr, [this]() {
```

Comment:
Does this fix https://improbableio.atlassian.net/browse/UNR-5085?

Reply:
Yeah this looks like it - didn't realise there was a tracking ticket.

Reply:
very minor nit - "all servers"